pkgs/by-name/gn/gnome-maps/package.nix

```diff
@@ -68,6 +69,13 @@ stdenv.mkDerivation (finalAttrs: {
     libsoup_3
   ];
 
+  mesonFlags = [
+    "--cross-file=${writeText "crossfile.ini" ''
+      [binaries]
+      gjs = '${lib.getExe gjs}'
```

Comment:
TODO: there should be an unified way to achieve this

Reply:
if anyone builds that i'll be happy to test/review!

a bit low on my own priorities to write that myself though